meteor/client/lib/multiSelect.tsx

```diff
@@ -6,13 +6,14 @@ import * as faCheckSquare from '@fortawesome/fontawesome-free-solid/faCheckSquar
 import * as faSquare from '@fortawesome/fontawesome-free-solid/faSquare'
 import * as faChevronUp from '@fortawesome/fontawesome-free-solid/faChevronUp'
 import * as FontAwesomeIcon from '@fortawesome/react-fontawesome'
+import { Manager, Reference, Popper } from 'react-popper'
 
 export interface MultiSelectEvent {
 	selectedValues: Array<string>
 }
 
 interface IProps {
-	availableOptions: _.Dictionary<string>
+	availableOptions: _.Dictionary<string | string[]>
```

Comment:
Would be good to have some description of how this dictionary of string arrays is supposed to work? It's not entirely clear to me how `availableOptions` can be a dictionary of string, so I think this needs some documentation of what's that supposed to mean.

Reply:
I added a comment describing the result of passing a `string[]` as a value in the Dictionary